src/test/regress/sql/alter_role_propagation.sql

```diff
@@ -78,15 +78,47 @@ SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlog
 SELECT run_command_on_workers($$SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlogin, rolreplication, rolbypassrls, rolconnlimit, rolpassword, EXTRACT (year FROM rolvaliduntil)) FROM pg_authid WHERE rolname = 'alter_role_1'$$);
 
 
--- table belongs to a role
+-- give login permissions so that we can connect and check if the previous queries were propagated
+ALTER ROLE alter_role_1 LOGIN CONNECTION LIMIT 10;
 
 
--- we don't support propagation of configuration_parameters and notice the users
-ALTER ROLE alter_role_1 SET enable_hashagg TO FALSE;
+-- alter configuration_parameter defaults for a user
+ALTER ROLE CURRENT_USER SET enable_hashagg TO FALSE;
+SELECT run_command_on_workers('SHOW enable_hashagg');
+
+-- reset to default values
+ALTER ROLE CURRENT_USER RESET enable_hashagg;
+SELECT run_command_on_workers('SHOW enable_hashagg');
+
```

Comment:
Users can come up with settings on the fly, e.g. `ALTER ROLE CURRENT_USER SET mysettings.test TO 'blah'`  will just work and can be read using `current_setting('mysettings.test')`.

It would be good to also add some nasty GUC names and values, e.g.:
`alter role "Me" set "app.dev""" to 'a\nb';`

(which seems to work fine)

We don't have to test this, but identifiers in PostgreSQL (role names and GUC names) can even contain end of line characters.

Reply:
Added this to the tests.